aries-site/src/data/featured.js

```diff
@@ -1,23 +1,14 @@
-import { Brush, Code, StarFill } from '@hpe-design/icons-grommet';
+import { Theme, Code } from '@hpe-design/icons-grommet';
 
 export const featured = [
   {
-    name: 'Designer guidance',
-    description: `Starter files to utilize our resources from HPE
-      Design System and HPE Brand.`,
-    icon: <Brush size="large" color="text-strong" />,
+    name: 'Design',
+    description: 'Access our Figma UI kits, components, and styles.',
+    icon: <Theme size="large" color="text-strong" />,
```

Comment:
These are decorative only, so they should not be read by screen reader.

```suggestion
    icon: <Theme size="large" color="text-strong"  aria-hidden="true" />,
```